cosmos-sdk-proto/Cargo.toml

```
@@ -18,7 +18,7 @@ rust-version = "1.72"
 [dependencies]
 prost = { version = "0.13", default-features = false }
 tendermint-proto = "0.39.1"
-serde = { version = "1.0.203", default-features = false }
+serde = { version = "1.0.203", default-features = false, features = ["alloc"] }
```

Hmm, it seems `protoc-gen-prost-serde` assumes traits like `ToString` are in the prelude. I guess to get this over the line, let's use the default `serde` features:

```suggestion
serde = "1.0.203"
```